tests/test_simple_x64.py

```diff
@@ -137,6 +137,9 @@ def test_assembler():
     CheckInstr(Add)('RAX', mem('[0x7fffffff]'))
     CheckInstr(Add)('RAX', -1)
 
+    # Registers
```

Comment:
Looks like the x64 instruction should be Pushfq & Popfq 
The DWORD equivalent cannot  be encoded in 64 bits mode but The intel manual match de capstone result for this one (present in manual intel : PUSHF/PUSHFD—Push EFLAGS Register onto the Stack)